hummingbot/connector/exchange/kraken/kraken_exchange.pyx

```diff
@@ -865,7 +865,12 @@ cdef class KrakenExchange(ExchangeBase):
 
         except Exception as e:
             self.c_stop_tracking_order(order_id)
-            order_type_str = 'LIMIT' if order_type is OrderType.LIMIT else "LIMIT_MAKER"
+            if order_type is OrderType.LIMIT:
+                order_type_str = 'LIMIT'
+            elif order_type is OrderType.LIMIT_MAKER:
+                order_type_str = 'LIMIT_MAKER'
+            else:
+                order_type_str = 'MAKER'
```

Comment:
Are you sure that Market is MAKER in kraken?

Reply:
good catch this should be `MARKET`. It's just used in the log message below.